core/basicobject/basicobject_spec.rb

```diff
@@ -76,7 +76,7 @@ def test_method() :test end
     obj.instance_variable_set(:@test, :value)
     obj.instance_variable_get(:@test).should == :value
 
-    obj.send(:hash).should be_an_instance_of(Fixnum)
+    lambda { obj.send(:hash) }.should_not raise_error
```

Comment:
You are right, the spec is just about the existence of `#hash`.
A `should respond_to(:hash)` would make a lot more sense then.